src/test/__snapshots__/tooltip-props.spec.js.snap

```diff
@@ -27,7 +27,7 @@ exports[`tooltip props clickable tooltip 1`] = `
     Lorem Ipsum
   </span>
   <div
-    class="react-tooltip undefined"
+    class="react-tooltip react-tooltip__place-top undefined"
```

Comment:
when using the `clickable` prop, we get an `undefined` classname when testing. this doesn't happen outside of testing.

any ideas @danielbarion?

Reply:
that's weird, I'll take a look when I have some available time, but this is not a blocker for this PR as this is an `extra-class`